conf/nautilus/rgw/tier_0_rgw.yaml

```diff
@@ -0,0 +1,42 @@
+globals:
+  - ceph-cluster:
+      name: ceph
+
+      node1:
+        role:
+          - installer
+          - mgr
+          - mon
+
+      node2:
+        disk-size: 20
+        no-of-volumes: 4
+        role:
+          - mgr
+          - osd
+
+      node3:
+        disk-size: 20
+        no-of-volumes: 4
+        role:
+          - mon
+          - osd
+
+      node4:
+        disk-size: 20
+        no-of-volumes: 4
+        role:
+          - mon
+          - osd
+
+      node5:
+        disk-size: 20
+        no-of-volumes: 4
+        role:
+          - osd
```

Comment:
Let's remove `osd` from this node and leave it as `rgw` node... primarily because we don't support co-location of `rgw` and `osd`. Though co-located services would work.